ai/debugging-with-ai.mdx

```diff
@@ -0,0 +1,85 @@
+---
+id: debugging-with-ai
+title: Debugging with AI
+description: Debugging with AI using the copy prompt button and fix with AI feature in VS Code.
+---
+import ProgressiveImage from '@theme/ProgressiveImage';
+import LiteYouTube from '@site/src/components/LiteYouTube';
+
+# Debugging with AI
+
+## Overview
+
+Debugging is a crucial part of the test development lifecycle, but it can often be time-consuming. With the power of AI, you can now accelerate the process of identifying and fixing test failures. This guide will show you how to leverage AI tools like GitHub Copilot directly within your Playwright workflow. You'll learn how to use features like "Copy Prompt" and "Fix with AI" to instantly get suggestions and fix failing tests from the Test Explorer, UI Mode, the HTML report, and the Trace Viewer, turning hours of debugging into minutes.
+
+<LiteYouTube
+  id="Ec_3h5Q8RHY"
+  title="Debug Playwright Tests with AI – Fix Failures Instantly"
+/>
+
+## 1. Copy Prompt (Faster Test Debugging)
+> (Image coming soon: Copy Prompt button in HTML report / trace viewer)
```

Comment:
<ProgressiveImage image={require('../images/getting-started/html-report-open.png')} alt="Copy Prompt button in HTML report / trace viewer" />